### ISSUE TITLE
Vita stick cant play games.

### ISSUE THREAD

**Issue by rdtor** [xerpi/vitastick#24]
The vita is recognised as game controller in the device and printer settings even the sticks are responsive in the properties menu. But when i launch game via Steam there is no response. Also incompatible with Play Station TV

**Comment by xerpi** [comment]
This means tou have to configure the game to use the sticks of the Vita. Games usually allow that remapping.

**Comment by rdtor** [comment]
is it possible to run with Playstation TV or are you working on it?

**Comment by xerpi** [comment]
It's not currently possible since the PSTV doesn't support generic USB gamepads. It would be possible by patching the PSTV OS with a plugin.

**Comment by rdtor** [comment]
Does there exist a plugin that can be patched on PSTV OS ?

**Comment by xerpi** [comment]
No, somebody has to program it. This is what should be used: https://github.com/vitasdk/vita-headers/blob/master/include/psp2/usbd.h

**Comment by rdtor** [comment]
Ok i will wait and see if the plugin comes for vita tv OS.

**Comment by Null-39** [comment]
Vixen could be the missing component to this
